packages/core/src/components/button/_button.scss

```diff
@@ -29,14 +29,37 @@
 
     &:not(:last-of-type) {
       margin-right: 1rem;
+
+      [dir='rtl'] & {
+        margin-right: 0;
+        margin-left: 1rem;
+      }
     }
 
-    &--pull-left {
+    // --pull-left is DEPRECATED in favor of --pull-start, and will be
+    // removed in a future release of Ray. Use of pull-left may result
+    // in confusing behavior for RTL language sensitive layouts.
```

Comment:
👏

Reply:
love these comments, thank you

Reply:
This class is undocumented and I plan on removing it all together anyway lol